ratelimit/cluster_test.go

```diff
@@ -1,3 +1,4 @@
+//go:build !race || redis
```

Comment:
I guess this should also be &&

Reply:
https://github.com/zalando/skipper/issues/1854